packages/atclient/include/atclient/atkeys.h

```diff
@@ -48,6 +50,7 @@ typedef struct atclient_atkeys {
 
   char *self_encryption_key_base64; // base64 encoded, AES-256 key, decrypted
 
+  char *apkam_symmetric_key_base64;
```

Comment:
if this is going to be the 5th thing inside of the struct, we should switch around so that

instead of this:
```h
#define ATCLIENT_ATKEYS_ENROLLMENT_ID_INITIALIZED (VALUE_INITIALIZED << 5)
#define ATCLIENT_ATKEYS_APKAM_SYMMETRIC_KEY_INITIALIZED (VALUE_INITIALIZED << 6)
```

we do this instead
```
#define ATCLIENT_ATKEYS_APKAM_SYMMETRIC_KEY_INITIALIZED (VALUE_INITIALIZED << 5)
#define ATCLIENT_ATKEYS_ENROLLMENT_ID_INITIALIZED (VALUE_INITIALIZED << 6)
```

so that we keep the order of things consistent

Reply:
That's a breaking change to the initialization vector, I only kept everything else the same to be consistent with the Dart ordering.